README.md

```diff
@@ -39,3 +39,15 @@ docker-compose down
 
 - [Building the backend](backend/README.md)
 - [Building the frontend](frontend/README.md)
+
+
+---
+
+### License
+
+<sup>
+Licensed under <a href="LICENSE">Apache License, Version 2.0</a>.
+Unless you explicitly state otherwise, any contribution intentionally submitted
+for inclusion in this project by you, as defined in the Apache-2.0 license,
+shall be licensed as above, without any additional terms or conditions.
+</sub>
```

Comment:
Mismatch between `<sub>` and `<sup`> which both look weird. Just use regular font. I would also rather put this into the contribution guide.

Reply:
```suggestion
Licensed under the [Apache License, Version 2.0](LICENSE).

Unless you explicitly state otherwise, any contribution intentionally submitted
for inclusion in this project by you
shall be licensed as above, without any additional terms or conditions.
```

Reply:
Will move to contribution guide once #7 is merged (or will change it later, in case this PR is merged before #7).